src/wasm-loader.js

```
@@ -0,0 +1,453 @@
+        const inputPtr = 0;
+        memoryArray.set(hexBytes, inputPtr);
+        const outputPtr = inputLen;
+
+        const hexLen = instance.exports.zig_keccak256_hex_old(inputPtr, inputLen, outputPtr);
+        const decoder = new TextDecoder();
+        return decoder.decode(new Uint8Array(memory.buffer, outputPtr, hexLen));
+      } else {
+        const hashBytes = keccak256Bytes(instance, input);
+        const bytesToHexFn = instance.bytesToHexOld || bytesToHexOld;
+        return bytesToHexFn(instance, hashBytes);
+      }
```

Biome (1.9.4)

[error] This else clause can be omitted because previous branches break early.

Unsafe fix: Omit the else clause.


(lint/style/noUselessElse)